### DISCUSSION TITLE
Export of settings

### DISCUSSION THREAD

**Discussion by guplem** [LesFerch/WinSetView#69] [Ideas]
I have just discovered this software, and I am lovin' it!

I like it so much that I would literally go and reconfigure each of the computers I interact with.

However, I haven't found any way of exporting my settings. 

I have been making a lot of trial and error and I can't remember everything I have changed...

Is the export of the configuration possible? 
Maybe they are saved in a file located in the folder next to the .exe or AppData files, and I can simply copy-paste them?

**Comment by LesFerch** [comment]
The settings are saved when you click Submit. If you're using Windows 10 or 11 the settings are saved with the exe in AppData\Win10.ini. Therefore, it's designed to be portable without any additional steps. If you run it from a flash drive, the settings will be saved on the flash drive. Or you could run it from a read/write network share and the settings will stay with the program. The exception is if you run it from a read-only location or a protected folder, such as "Program Files". In that case, the settings are saved to your profile's %appdata% folder. You can also enable the Advanced interface and use Save Settings to save a settings file wherever you like and conversely use Load Settings to load a settings file.

**Reply by LesFerch** [reply]
I should have also mentioned that the Win10.ini file is automatically loaded the next time you run the program.

**Reply by guplem** [reply]
Thanks! That's amazing!

Thank you very much for all the info.

Maybe this should be part of the documentation? I believe it isn't - or at least I didn't see it.

Thanks again!

**Reply by LesFerch** [reply]
Yes, I see there is room for improvement in the documentation in regards to this topic. Information about the saved settings is currently spread out in three places. I'll review and revise. Thanks!
